modules/cnv-cdrom-wizard-fields-web.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv-create-vms.adoc
+// * cnv/cnv_vm_templates/cnv-creating-vm-template.adoc
+
+[id="cnv-cdrom-wizard-fields-web_{context}"]
+= CD-ROM fields
+
+|===
+|Media Type | Description
```

Comment:
in wizard this is actually called "Media Type"

Reply:
@jelkosz In the defect that Guohua noted (shown above in this PR), it was mentioned that the GUI for Wizard and the Virtual Machine page is displayed as follows:

Wizard
Source > Container / URL / Attach Disk

VM Details Page
Media Type > Container / URL / Persistent Volume

Therefore, I updated the documentation to reflect the UI notes that Guohua documented in the defect. 

Please advise should I be changing the documentation to show:

Wizard >
Media Type > Container / URL / Attach Disk

Please confirm?

Reply:
no, you are correct, it has to be "source" here, so it is correct as is.